pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -966,9 +966,15 @@ func GetOpenshiftBootstrapClusterRoleBindings() []rbac.ClusterRoleBinding {
 		newOriginClusterBinding(StatusCheckerRoleBindingName, StatusCheckerRoleName).
 			Groups(AuthenticatedGroup, UnauthenticatedGroup).
 			BindingOrDie(),
-		newOriginClusterBinding(NodeRoleBindingName, NodeRoleName).
-			Groups(NodesGroup).
-			BindingOrDie(),
+		// Preserve the empty node binding with no subjects for tightening reconciliation
+		rbac.ClusterRoleBinding{
+			ObjectMeta: metav1.ObjectMeta{Name: NodeRoleBindingName},
+			RoleRef: rbac.RoleRef{
+				APIGroup: rbac.GroupName,
+				Kind:     "ClusterRole",
+				Name:     NodeRoleName,
+			},
+		},
```

Comment:
please use newOriginClusterBinding() or even better rbac.newClusterBinding() if you do not have to maintain a name difference between the role and the binding.

Reply:
that helper does not allow empty subjects, which we want here.

Reply:
very annoying, what is the purpose of a binding with no subjects ?

Reply:
so that tightening reconciliation removes the grant to the `system:nodes` group